### PR TITLE
MB-12540: Integration tests for PPM expense submission

### PR DESCRIPTION
**Pull request by kylehilltruss**
## Summary

This pull request completes the integration tests for PPM expenses submission, including the since-added capability of uploading a receipt document and verifying that a newly-created or edited expense record saves correctly.

## Setup to Run Your Code

Run the integration test client locally:

```sh
make e2e_test_fresh
```

### Additional steps

In an instance running on your machine, run the `mymove/expenses.js` integration test suite. Verify that all four tests in that suite pass.

Alternatively, you can view the `integration_tests_milmove` workflow step on CircleCI for this pull request and verify that these (and all other) tests pass.

## Verification Steps for Author

These are to be checked by the author.

- [ ] Request review from a member of a different team.
- [ ] Have the Jira acceptance criteria been met for this change?

### CODE REVIEW

File: cypress/integration/milmove/ppms/expenses.js [hunk 1]
```diff
@@ -11,6 +10,7 @@ describe('Expenses', function () {
   });
   beforeEach(() => {
     cy.intercept('GET', '**/internal/moves/**/mto_shipments').as('getShipment');
+    cy.intercept('POST', '**/internal/uploads**').as('uploadFile');
```
Comment (ronaktruss) [inline cypress/integration/milmove/ppms/expenses.js:13]: I think part of the intent of this story was to update the UI to leverage the new API that we created as part of https://github.com/transcom/mymove/pull/9351 and to test that new API here ([ref](https://dp3.atlassian.net/browse/MB-12540?focusedCommentId=24174)).  The changes should be fairly similar to the ones made in that PR ([ref](https://github.com/transcom/mymove/pull/9351/files#diff-f8775a9096163b4705216218847b143c26d93082be9d1c441d325d3f9846b439))

**Review by pearl-truss — COMMENTED** [review]
I might be mistaken but I think given the AC there should be a non happy path test for when the required fields are missing and another for when the data is invalid

**Comment by kylehilltruss** [comment]
> I might be mistaken but I think given the AC there should be a non happy path test for when the required fields are missing and another for when the data is invalid

Good catch -- as we discussed in standup today, we've now removed that from the AC for this ticket (in lieu of existing front-end unit tests that control for the same)

File: cypress/integration/milmove/ppms/expenses.js (hunk 1 above)
Comment (kylehilltruss) [reply cypress/integration/milmove/ppms/expenses.js]: Updated accordingly

File: cypress/integration/milmove/ppms/expenses.js [hunk 2]
```diff
@@ -65,6 +75,13 @@ describe('Expenses', function () {
 
       cy.get('input[name="missingReceipt"]').as('missingReceipt').should('not.be.checked');
       cy.get('@missingReceipt').click({ force: true });
+
+      cy.get('button').contains('Save & Continue').should('be.enabled').click();
+      cy.location().should((loc) => {
+        expect(loc.pathname).to.match(/^\/moves\/[^/]+\/shipments\/[^/]+\/review/);
+      });
+
+      cy.contains('PA Turnpike EZ-Pass');
```
Comment (ronaktruss) [inline cypress/integration/milmove/ppms/expenses.js:84]: Should we also verify if `Tolls` and `54.32` are displaying since those values are also updated?

**Review by ronaktruss — COMMENTED** [review]
LGTM. Left a comment about potentially adding 2 more quick checks on the edit scenario to ensure we see the updated values.

**Review by ronaktruss — APPROVED** [review]

**Pull request merged**